go/libraries/doltcore/sqle/enginetest/dolt_queries_diff.go

```diff
@@ -528,14 +528,44 @@ var DiffSystemTableScriptTests = []queries.ScriptTest{
 			},
 			{
 				Query:    "SELECT COUNT(*) FROM DOLT_DIFF_t;",
-				Expected: []sql.Row{{1}},
+				Expected: []sql.Row{{7}},
 			},
 			{
 				Query:    "SELECT to_pk, to_c1, from_pk, from_c1, diff_type FROM DOLT_DIFF_t where to_commit=@Commit4;",
 				Expected: []sql.Row{{7, 8, nil, nil, "added"}},
 			},
 		},
 	},
+	{
+		// Similar to previous test, but with one row to avoid ordering issues.
+		Name: "altered keyless table add pk with removed", // https://github.com/dolthub/dolt/issues/8625
```

Comment:
I didn't understand the "with removed", so suggesting pulling it out

```suggestion
		Name: "altered keyless table add pk", // https://github.com/dolthub/dolt/issues/8625
```